meson.build:78:11: ERROR: Dependency lookup for neatvnc with method 'pkgconfig' failed: Invalid version, need 'neatvnc' ['<0.9.0'] found '0.9.0'.

--- meson.build.orig	2024-02-25 11:22:05 UTC
+++ meson.build
@@ -57,7 +57,7 @@ aml_version = ['>=0.3.0', '<0.4.0']
 jansson = dependency('jansson')
 
 aml_version = ['>=0.3.0', '<0.4.0']
-neatvnc_version = ['>=0.8', '<0.9.0']
+neatvnc_version = ['>=0.8', '<0.10.0']
 
 neatvnc_project = subproject(
 	'neatvnc',
